scripts/exregional_get_extrn_mdl_files.sh

```diff
@@ -171,6 +182,86 @@ ${cmd}
 #
 #-----------------------------------------------------------------------
 #
+# Merge GEFS files
+#
+#-----------------------------------------------------------------------
+#
+function filename_mod() {
+    # Function that modifies the filenames
+    fn_mod=$(echo "$1" |
+             sed "s|{mem:02d}|$num|g" |
+             sed "s|t{hh}|t${hh}|g" |
+             sed "s|f{fcst_hr:03d}|f`printf %03d $fcst_hr`|g" |
+             sed "s|f{fcst_hr:02d}|f`printf %02d $fcst_hr`|g" )
+    echo $fn_mod
+}
```

Comment:
This use of `sed` will likely break functionality on a mac. Not sure if that matters here?

Reply:
That function will go away with my latest changes. But I do have a sed command later in the script, which I changed to 'awk' instead.